packages/rocketchat-lib/server/lib/sendEmailOnMessage.js

```diff
@@ -22,14 +22,30 @@ RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
 		}
 
 		emailSubject = TAPi18n.__('Offline_Mention_Email', {
-			site: RocketChat.settings.get('Site_Name'),
 			user: message.u.username,
 			room: room.name
 		});
 	}
 
-	// code duplicate of packages/rocketchat-ui-message/message/message.coffee
-	message.html = s.escapeHTML(message.msg);
+	var getMessageLink = () => {
+		var path = directMessage ? `direct/${ room.username }` : `channel/${ room.name }`;
+		var style = [
+			'color: #fff;',
+			'padding: .5em;',
+			'background-color: #04436a;',
+			'display: block;',
+			'width: 10em;',
+			'text-align: center;',
+			'text-decoration: none;',
+			'margin: auto;',
+			'margin-bottom: 8px;'
+		].join(' ');
+		return `<a style="${ style }" href="${ process.env.ROOT_URL }${ path }">GO TO MESSAGE</a>`;
```

Comment:
@danielrohers can you change this to use TAPi18n to `GO TO MESSAGE` and add this text to en.i18n.json?